code/game/objects/items/weapons/tech_disks.dm

```diff
@@ -1,18 +1,34 @@
 /obj/item/disk
-	icon = 'icons/obj/cloning.dmi'
-	icon_state = "datadisk2"
-	item_state = "card-id"
-	w_class = ITEM_SIZE_SMALL
-	matter = list(/decl/material/solid/plastic = 30, /decl/material/solid/metal/steel = 30, /decl/material/solid/glass = 10)
+	name = "data disk"
+	icon = 'icons/obj/items/device/diskette.dmi'
+	icon_state = ICON_STATE_WORLD
+	w_class = ITEM_SIZE_TINY
+	material = /decl/material/solid/plastic
+	matter = list(/decl/material/solid/metal/steel = MATTER_AMOUNT_TRACE)
+	var/label = "label_blank"
+	
+/obj/item/disk/on_update_icon()
+	. = ..()
```

Comment:
Not much point calling down the tree if you cut overlays right after.

Reply:
well in case someone needs material alteration stuff down the line.